Empire/Empire.cpp

```diff
@@ -2930,7 +2930,7 @@ void Empire::CheckProductionProgress() {
 
         // consume the item's special and meter consumption
         std::map<std::string, std::map<int, float> > sc = elem.item.CompletionSpecialConsumption(elem.location);
-        for (std::map<std::string, std::map<int, float>>::iterator sc_it = sc.begin(); sc_it != sc.end(); ++sc_it) {
+        for (std::map<std::string, std::map<int, float> >::iterator sc_it = sc.begin(); sc_it != sc.end(); ++sc_it) {
```

Comment:
maybe you should submit this as a separate pull request, rather than including it in multiple unrelated ones?